Mage/src/main/java/mage/abilities/TriggeredAbilityImpl.java

```diff
@@ -436,10 +436,7 @@ public static boolean isInUseableZoneDiesTrigger(TriggeredAbility source, GameEv
         if (game.getState().getZone(source.getSourceId()) == Zone.BATTLEFIELD) {
             sourceObject = game.getPermanent(source.getSourceId());
         } else {
-            // TODO: multiple calls of ApplyEffects all around the code are breaking a short living lki idea
```

Comment:
That’s all still actual (deleted todo and examples). See another comment above.

Reply:
Nope, I don't see how it's actual. ApplyEffects no longer breaks short living LKI.